lib/GADS/Schema/Result/User.pm

```diff
@@ -1226,4 +1228,23 @@ sub export_hash
     };
 }
 
+has encryption_key => (
+    is      => 'lazy',
+);
+
+sub _build_encryption_key {
+    my $self = shift;
+    
+    my $header_json  = '{"typ":"JWT","alg":"HS256"}';
+    # This is a string because encode_json created the JSON string in an arbitrary order and we need the same key _every time_!
```

Comment:
There should be options in the JSON-encoder to order the keys, but happy to leave it like this for the time being.